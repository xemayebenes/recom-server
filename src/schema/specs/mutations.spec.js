import Mutations from '../mutations';

describe('Mutations', () => {
    it('should match snapshot', () => {
        expect(Mutations).toMatchSnapshot();
    });

});