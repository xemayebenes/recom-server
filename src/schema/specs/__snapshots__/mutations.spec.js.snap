// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mutations should match snapshot 1`] = `
"
  type Mutation {
    addMovie(externalId: Int!): UserMovie
    addSerie(externalId: Int!): UserSerie
  }
"
`;
